src/goto-instrument/cover.cpp

```diff
@@ -28,21 +28,26 @@ Date: May 2016
 void instrument_cover_goals(
```

Comment:
Since this function has so many arguments, could they be packed into an `instrument_cover_goals_infot` struct?